lib/logging.properties

```diff
@@ -23,3 +23,5 @@ com.agafua.syslog.SyslogHandler.escapeNewlines = false
 
 # to disable double timestamps in syslog uncomment next line
 #net.java.sip.communicator.util.ScLogFormatter.disableTimestamp=true
+
+org.jitsi.xmpp.mucclient.MucClientManager.level=SEVERE
```

Comment:
Is this to silence "Setting a presence extension" logs?

Reply:
Exactly

Reply:
Should we move that log to a lower level instead?

Reply:
We could do it in addition. This PR targets `jvb_1.0` and can't use an updated jicofo (unless we maintain a branch, but that would be a lot of hassle to silence some logs)